imap_processing/codice/codice_l1a.py

```diff
@@ -37,6 +35,7 @@
 from imap_processing.utils import group_by_apid, sort_by_time
 
 logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
```

Comment:
@greglucas cleaned up logger at one point. Could it be because logging uses `INFO` by default?

Reply:
I think this can be removed. I put it in when I was messing around with trying to get log messages printed to the screen during a local `pytest` run (I still haven't found a good solution for that 🤷🏻)

Reply:
When I am running test locally, I write print statement and then run the pytest with `-s` parameter. It will print all the print statements. Eg.
```
pytest imap_processing/tests/swapi -s
```

Reply:
Adding this in our `test/conftest.py` did the trick for me. But you still have to add `-s` to see the log statement.
```
def pytest_configure(config):
    # Set up logging to output to the console at DEBUG level
    logging.basicConfig(level=logging.DEBUG, format='%(asctime)s - %(levelname)s - %(message)s')

    # If you know the specific logger used by your application, set it directly:
    logger = logging.getLogger(__name__)
    logger.setLevel(logging.INFO)
```

Reply:
Thanks for sharing, I will try it out!